src/Main.elm

```diff
@@ -314,23 +344,40 @@ view ({ error, attempts, word, currentTry } as model) =
                             []
                         , button [ class "btn btn-primary" ] [ text "Submit" ]
                         ]
-                    , div [ class "form-text" ] [ text "Enter a 5 letters English word" ]
+                    , div [ class "form-text" ] [ text "Enter a 5 letters ", text (toString language), text " word" ]
                     ]
         ]
 
 
 main : Program () Model Msg
 main =
     Browser.element
-        { init = init
+        { init = init English
```

Comment:
We could probably detect users language from their browser environment, eg by using `navigator.language` and passing it to Flags, so we would preselect their language by default!